packages/gestalt/src/SheetMobile/FullPage.js

```diff
@@ -61,8 +62,25 @@ export default function FullPage({
   showDismissButton,
   subHeading,
 }: Props): Node {
+  const id = useId();
+
+  // Consumes DefaultLabelProvider
   const { accessibilityLabel: defaultAccessibilityLabel } = useDefaultLabelContext('SheetMobile');
 
+  // Consumes GlobalEventsHandlerProvider
+  const { sheetMobile } = useGlobalEventsHandlerContext() ?? {
+    sheetMobile: { onOpen: () => {}, onClose: () => {} },
+  };
+
+  useEffect(() => {
+    sheetMobile?.onOpen?.();
```

Comment:
This reads a little wonky. Consider renaming the variable from useGlobalEventsHandlerContext to something like "sheetMobileHandlers" so this reads more like `sheetMobileHandlers?.onOpen?.()`

Reply:
DONE